microGallery_Android/app/src/main/kotlin/studio/lunabee/amicrogallery/photoviewer/PhotoViewerUiState.kt

```diff
@@ -3,6 +3,7 @@ package studio.lunabee.amicrogallery.photoviewer
 import studio.lunabee.compose.presenter.PresenterUiState
 import studio.lunabee.microgallery.android.data.Picture
 
-data class PhotoViewerUiState( // picture is not here at the beginning, it must be found in db
+data class PhotoViewerUiState(
+    // picture is not here at the beginning, it must be found in db
```

Comment:
What we usually do is have several state,
idle -> for when the data is loading from db
PhotoViewerUiState -> when it's loaded with picture not null
error/empty or navigateBack -> if picture fail to load